Assets/LeapMotionModules/ElementRenderer/Scripts/Features/Mesh/ProceduralMeshes/ProceduralPanel.cs

```diff
@@ -118,4 +175,9 @@ public class ProceduralPanel : ProceduralMeshSource {
       return (dv / (vertCount - 1.0f)) * size;
     }
   }
+
+  public enum ResolutionType {
+    Vertices,
+    VerticesPerMeter
```

Comment:
Vertices per meter was a little unintuitive until I recognized that it was vertices per _rectilinear_ meter (ideally it would be "vertices per radian/degree" but I recognize this makes it dependent on the base space and its position within it so vertices per rectilinear meter is a fine approximation)